test/e2e/reconciler/ingress/controller_test.go

```diff
@@ -158,30 +153,19 @@ func TestIngressController(t *testing.T) {
 				metav1.GroupResource{Group: apiresource.GroupName, Resource: "negotiatedapiresources"},
 			)
 			require.NoError(t, err)
-			kubefixtures.Create(t, sourceCrdClient.ApiextensionsV1().CustomResourceDefinitions(),
-				metav1.GroupResource{Group: "core.k8s.io", Resource: "services"},
-				metav1.GroupResource{Group: "apps.k8s.io", Resource: "deployments"},
-				metav1.GroupResource{Group: "networking.k8s.io", Resource: "ingresses"},
-			)
 
 			resources := sets.NewString("ingresses.networking.k8s.io", "deployments.apps", "services")
-			syncerFixture := framework.NewSyncerFixture(t, resources, source, orgClusterName, clusterName)
-
-			sink := syncerFixture.RunningServer
-			sinkConfig := sink.DefaultConfig(t)
-			sinkCrdClient, err := apiextensionsclientset.NewForConfig(sinkConfig)
-			require.NoError(t, err, "failed to create apiextensions client")
-
-			// TODO(jmprusi): Remove this one once Kind e2e is a thing.
-			t.Logf("Installing test CRDs into sink cluster...")
-			kubefixtures.Create(t, sinkCrdClient.ApiextensionsV1().CustomResourceDefinitions(),
-				metav1.GroupResource{Group: "core.k8s.io", Resource: "services"},
-				metav1.GroupResource{Group: "apps.k8s.io", Resource: "deployments"},
-				metav1.GroupResource{Group: "networking.k8s.io", Resource: "ingresses"},
-			)
-			require.NoError(t, err)
-
-			t.Log("Starting syncer")
+			syncerFixture := framework.NewSyncerFixtureWithCRDs(t, resources, source, orgClusterName, clusterName, func(sinkConfig *rest.Config) {
+				sinkCrdClient, err := apiextensionsclientset.NewForConfig(sinkConfig)
+				require.NoError(t, err, "failed to create apiextensions client")
+				t.Logf("Installing test CRDs into sink cluster...")
+				err = configcrds.CreateFromFS(ctx, sinkCrdClient.ApiextensionsV1().CustomResourceDefinitions(), embeddedResources,
```

Comment:
Will need to use kubefixtures.Create (as per line 177) - the embedded resources are now shared with other tests.